spec/file-icons-spec.coffee

```diff
@@ -21,3 +25,45 @@ describe 'FileIcons', ->
     FileIcons.resetService()
 
     expect(FileIcons.getService()).not.toBe(service)
+
+  
+  describe 'Class handling', ->
```

Comment:
I think that file is meant to test `FileIcons` without involving the UI, and just assert that it can be configured properly. This one seems more like an integration test that verifies that the tree-view package is able to deal with arrays (and strings) coming from a service.